components/FormInput/FormInput.module.css

```diff
@@ -0,0 +1,28 @@
+.labelForm {
+    width: 300px;
```

Comment:
Usually, fixed width/height in pixels could cause issues on responsive designs, please use % on the widths and instead of height, use paddings (and please, use the css vars